packages/peregrine/lib/talons/CartPage/GiftOptions/__tests__/useGiftOptions.spec.js

```diff
@@ -53,6 +53,7 @@ jest.mock('@magento/peregrine/lib/context/cart', () => {
 });
 
 jest.mock('lodash.throttle', () => {
+    // eslint-disable-next-line
     return (callback, delay, config) => {
```

Comment:
You can just remove these extra props; `jest.mock('lodash.throttle', () => callback => callback);`